real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/LoggingPanel.java

```diff
@@ -23,6 +27,8 @@
  * {@link RobobuggyGUIContainer} used for logging information
  */
 public class LoggingPanel extends RobobuggyGUIContainer{
+	//TODO: test to make sure all these dont need to be static
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'resetBtn' could be replaced by a local variable.](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2230250215/issues/source?bid=2938665&fileBranchId=3172754#l31)